rest/handler.go

```diff
@@ -953,6 +953,7 @@ func (h *handler) checkPublicAuth(dbCtx *db.DatabaseContext) (err error) {
 				provider := dbCtx.Options.OIDCOptions.Providers.GetProviderForIssuer(h.ctx(), issuerUrlForDB(h, dbCtx.Name), testProviderAudiences)
 				if provider != nil && provider.ValidationKey != nil {
 					if base.StringDefault(provider.ClientID, "") == username && *provider.ValidationKey == password {
+						auditFields = base.AuditFields{base.AuditFieldAuthMethod: "bearer"}
```

Comment:
Probably should be considered basic auth given username/password used. But at least this is effectively test-only code.

```suggestion
						auditFields = base.AuditFields{base.AuditFieldAuthMethod: "basic"}
```

Reply:
I wasn't thinking more than oidc -> bearer in my head.